error_valid_check.py

```diff
@@ -0,0 +1,104 @@
+import string
+
+class ErrorsCheck:
+    def __init__(self, phone_number_str=None, email_address_str=None):
+        self.phone_number_str = phone_number_str
+        self.email_address_str = email_address_str
+        self.errors = []
+
+    # Проверка номера телефона
+    def validate_phone_number(self):
+        if self.phone_number_str is None:
```

Comment:
```suggestion
        if not self.phone_number_str:
```
Такий вираз більш "pythonic"